app/frontend/src/components/Answer/Answer.tsx

```diff
@@ -23,11 +29,16 @@ interface Props {
     showFollowupQuestions?: boolean;
     showSpeechOutputBrowser?: boolean;
     showSpeechOutputAzure?: boolean;
-    speechUrl: string | null;
 }
 
 export const Answer = ({
     answer,
+    index,
```

Comment:
Hm so it feels a little awkward that we have to send so many speech-specific properties to Answer. Ideally it could feel a bit more modular.
Would it be better if there was a SpeechConfig constructed in Chat.tsx/Ask.tsx, and that was passed to Answer and then to the component?

Reply:
Sure let me see what I can do.

Reply:
Done.
It looks way much better now, thanks!